pallets/lbp/src/lib.rs

```diff
@@ -584,6 +609,10 @@ pub mod pallet {
 			T::MultiCurrency::transfer(asset_a, &pool_id, &who, amount_a)?;
 			T::MultiCurrency::transfer(asset_b, &pool_id, &who, amount_b)?;
 
+			if Self::collected_fees(&pool_data) > 0 {
+				T::MultiCurrency::remove_lock(COLLECTOR_LOCK_ID, asset_a, &pool_data.fee_collector)?;
```

Comment:
A user can create two pools collecting the same asset. This removes the lock created by another pool.

Reply:
good point, we should restrict you to to just one pool at time, otherwise we would have do do fee balance accounting in the LBP pallet itself